ee/vellum_ee/workflows/display/nodes/vellum/utils.py

```diff
@@ -97,5 +99,11 @@ def create_pointer(
     vellum_variable_value = primitive_to_vellum_value(value)
     if pointer_type is InputVariablePointer:
         return InputVariablePointer(type="INPUT_VARIABLE", data=InputVariableData(input_variable_id=value))
+    elif pointer_type is WorkspaceSecretPointer:
+        return WorkspaceSecretPointer(
+            type="WORKSPACE_SECRET", data=WorkspaceSecretData(type="STRING", workspace_secret_id=value)
+        )
+    elif pointer_type is ExecutionCounterPointer:
+        return ExecutionCounterPointer(type="EXECUTION_COUNTER", data=ExecutionCounterData(node_id=value))
     else:
```

Comment:
nit: I suggest looking for usages of `assertUnreachable`. It'd be cool to ensure we're covering all our bases here and throwing exceptions if we come across data we shouldn't see

Reply:
Is there a python equivalent? I am trying to look for one but don't see it unless you meant for me to include one for our python utils

Reply:
Oh woops no. But we should probs still raise a `ValueError` in our `else` block if we come across something we don't expect.